packages/core/src/scrim/Scrim.tsx

```diff
@@ -92,6 +92,14 @@ export interface ScrimProps extends HTMLAttributes<HTMLDivElement> {
    * To maintain open and close of the scrim.
    */
   open?: boolean;
+  /**
+   * Set to true to enable the container use case. If true the Scrim will be bound to a container ref you pass via the `parentRef` prop.
+   * Default value is false, and the default behavior is for Scrim to be bound to the document viewport.
+   */
+  enableContainerMode?: boolean;
+  /**
+      * Prop necessary to enable container use case. Pass the parent element ref that you want Scrim to be bound to.
+      * Default value is undefined, and the default behavior is for Scrim to be bound to the document viewport.
   /**
    * Prop to enable container use case. Pass the parent element ref that you want Scrim to be bound to.
```

Comment:
Think you need to to remove this line? Looks like it's duplicated on line 101

Reply:
Good catch